hack/make.sh

```diff
@@ -16,7 +16,7 @@ function build_test ()
 
 	# build pouch binaries and execute unit tests.
 	echo "Build pouch binaries and execute unit tests."
-	docker run -ti -v `pwd`:/go/src/github.com/alibaba/pouch pouch:test bash -c "make check && make build && make unit-test"
+	docker run -ti -v `pwd`:/go/src/github.com/alibaba/pouch pouch:test bash -c "make check && make build && make unit-test && cd test && go test -c -o integration-test"
```

Comment:
Could we add an entrypoint in Makefile?

Reply:
Yes, I can.  I will create another pull request to add binary, unittest, integration test.

Reply:
OK